lib/pxconfig.js

```diff
@@ -156,11 +196,48 @@ function pxInternalConfig() {
         CE_DIGEST: 'sha256',
         CE_ALGO: 'aes-256-cbc',
 
-        STATIC_FILES_EXT: ['.css', '.bmp', '.tif', '.ttf', '.docx', '.woff2', '.js', '.pict', '.tiff', '.eot', '.xlsx', '.jpg', '.csv', '.eps', '.woff', '.xls', '.jpeg', '.doc', '.ejs', '.otf', '.pptx', '.gif', '.pdf', '.swf', '.svg', '.ps', '.ico', '.pls', '.midi', '.svgz', '.class', '.png', '.ppt', '.mid', 'webp', '.jar'],
+        STATIC_FILES_EXT: [
+            '.css',
+            '.bmp',
+            '.tif',
+            '.ttf',
+            '.docx',
+            '.woff2',
+            '.js',
+            '.pict',
+            '.tiff',
+            '.eot',
+            '.xlsx',
+            '.jpg',
+            '.csv',
+            '.eps',
+            '.woff',
+            '.xls',
+            '.jpeg',
+            '.doc',
+            '.ejs',
+            '.otf',
+            '.pptx',
+            '.gif',
+            '.pdf',
+            '.swf',
+            '.svg',
+            '.ps',
+            '.ico',
+            '.pls',
+            '.midi',
+            '.svgz',
+            '.class',
+            '.png',
+            '.ppt',
+            '.mid',
+            'webp',
+            '.jar',
+        ],
 
         MONITOR_MODE: {
             MONITOR: 0,
-            BLOCK: 1
+            BLOCK: 1,
```

Comment:
The last value of JSON should not have ","

Reply:
according to esLint it does